sql-files/item_db_re.sql

```diff
@@ -2794,7 +2794,7 @@ REPLACE INTO `item_db_re` VALUES (4561,'Professor_Card','Professor Celia Card',6
 REPLACE INTO `item_db_re` VALUES (4562,'Champion_Card','Champion Chen Card',6,20,NULL,10,NULL,NULL,NULL,NULL,NULL,NULL,NULL,16,NULL,NULL,NULL,NULL,'bonus2 bAddClass,Class_All,(readparam(bAgi)>=110)?14:7; bonus5 bAutoSpellWhenHit,"MO_INVESTIGATE",5,70,BF_WEAPON,1;',NULL,NULL);
 REPLACE INTO `item_db_re` VALUES (4563,'Creator_Card','Creator Flamel Card',6,20,NULL,10,NULL,NULL,NULL,NULL,NULL,NULL,NULL,16,NULL,NULL,NULL,NULL,'bonus bCritical,(readparam(bStr)>=110)?40:20; bonus5 bAutoSpellWhenHit,"AM_ACIDTERROR",5,70,BF_WEAPON,1;',NULL,NULL);
 REPLACE INTO `item_db_re` VALUES (4564,'Stalker_Card','Stalker Gertie Card',6,20,NULL,10,NULL,NULL,NULL,NULL,NULL,NULL,NULL,16,NULL,NULL,NULL,NULL,'bonus bHit,(readparam(bLuk)>=110)?40:20; bonus5 bAutoSpellWhenHit,"ST_FULLSTRIP",1,70,BF_WEAPON,1;',NULL,NULL);
-REPLACE INTO `item_db_re` VALUES (4565,'Paladin_Card','Paladin Randel Card',6,20,NULL,10,NULL,NULL,NULL,NULL,NULL,NULL,NULL,16,NULL,NULL,NULL,NULL,'bonus bMaxHPrate,(readparam(bLuk)>=110)?20:10; bonus5 bAutoSpellWhenHit,"CR_GRANDCROSS",10,70,BF_WEAPON,0;',NULL,NULL);
+REPLACE INTO `item_db_re` VALUES (4565,'Paladin_Card','Paladin Randel Card',6,20,NULL,10,NULL,NULL,NULL,NULL,NULL,NULL,NULL,16,NULL,NULL,NULL,NULL,'bonus bMaxHPrate,(readparam(bInt)>=110)?20:10; bonus5 bAutoSpellWhenHit,"CR_GRANDCROSS",10,70,BF_WEAPON,0;',NULL,NULL);
```

Comment:
No need for this. It will be automatically updated by rathenaAPI when this get merged.

Reply:
Thanks for the clarification.